python/tank/commands/setup_project_params.py

```diff
@@ -839,8 +839,11 @@ def __init__(self, config_uri, sg, log):
         # load the required storage roots for the config
         self._storage_roots = StorageRoots.from_config(self._cfg_folder)
 
-        # populate any missing defaults in the file
-        self._storage_roots.populate_defaults()
+        # if the file doesn't exist, create some defaults. if the file does
+        # exist, we'll assume that the intention is not to define any storage
+        # roots, as is the case with the basic and site config.
```

Comment:
This comes from a fix @josh-t made that was necessary to have the old-style side config work in the integration test. Please ignore as it will be reviewed and merged separately.